index.html

```diff
@@ -4,231 +4,178 @@
-          </ul>
-        </div>
-      </nav>
-    <!-- mobile home section -->
-    <div class="header-background">
-      <h1 class="primary-text">I'm ndikumana <br>Isaie to see you</h1>
-      <p class="primary-text2">
-        I am a software developer! I can help you build a product , feature or
-        website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, do not hestiate to contact
-        me.
-      </p>
-      <div class="connect">
-        <p>Let's connect</p>
-      </div>
-      <div class="social-media">
-        <ul>
-          <li>
-            <a href="https://github.com/ndikumanaisaie">
-              <img src="./images/github.svg" alt="github" />
-            </a>
-          </li>
-          <li>
-            <a href="https://twitter.com/Ndikuma38670724">
-              <img src="./images/medium.svg" alt="medium" />
-            </a>
-          </li>
-          <li>
-            <a href="https://twitter.com/Ndikuma38670724">
-              <img src="./images/twitter.svg" alt="twitter" />
-            </a>
-          </li>
-          <li>
-            <a href="vector">
-              <img src="./images/Vector.svg" alt="" />
-            </a>
-          </li>
-          <li>
-            <a href="https://www.linkedin.com/in/ndikumana-isaie-21166273/">
-              <img src="./images/Linkedin.svg" alt="linkedIn" />
-            </a>
-          </li>
-        </ul>
+    <div class="main">
+      <div class="bg-color">
+        <header class="text-white font-lato">
+          <div class="hide-on-sm show-on-lg ">
+            <div class="bg-gray-color flex items-center">
+              <nav class="subHeaderStyle">
+                <ul class="header-top">
+                  <li class="subHeaderListStyle"><i class="fab fa-facebook-f fb"></i></li>
+                  <li class="subHeaderListStyle"><i class="fab fa-twitter twt"></i></li>
+                  <li class="subHeaderListStyle">English</li>
+                  <li class="subHeaderListStyle">My Page</li>
+                  <li class="subHeaderListStyle">Logout</li>
+                </ul>
+              </nav>
+            </div>
+            <div class="header-bottom">
+              <nav class="main-top">
+                <a class="cursor-pointer" href="index.html"><img class="site-logo" src="./images/logo1.png" alt="site logo on header "/></a>
+                <ul class="main-head flex items-center">
+                  <li class="subHeaderListStyle"><a href="about.html" class="no-underline gray-text-color">About</a></li>
+                  <li class="subHeaderListStyle"><a href="#" class="no-underline gray-text-color">Program</a></li>
+                  <li class="subHeaderListStyle"><a href="#" class="no-underline gray-text-color">Join</a></li>
+                  <li class="subHeaderListStyle"><a href="#" class="no-underline gray-text-color">Sponsor</a></li>
```

Comment:
- [ ] Please kindly ensure to stick to the design guidelines(margins, padding, spacing, font, color etc). In this case, your top header does not have the proper right padding that the original design has. The bottom header does not have the proper padding around it like the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/169177012-90e3b53c-f9e4-4a25-bbea-95dc7c7b696f.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/169177072-74aced2f-6243-4b2d-9f4e-72e27fd34c0c.png)

Reply:
- [ ] As requested by the previous reviewer, kindly ensure you add the correct right padding to the top header as you can see there is no padding in the image below

![image](https://user-images.githubusercontent.com/90556221/169279234-1ba2a134-0065-43c9-88ca-d7cde81a7864.png)